templates/vsphere/template-variables.rc

```diff
@@ -0,0 +1,36 @@
+# Source: https://github.com/kubernetes-sigs/cluster-api-provider-vsphere/blob/main/docs/getting_started.md
+
+export VSPHERE_USERNAME=""                                                      # The username used to access the remote vSphere endpoint
+export VSPHERE_PASSWORD=""                                                      # The password used to access the remote vSphere endpoint
+
+export VSPHERE_SERVER="10.0.0.1"                                                # The vCenter server IP or FQDN
+export VSPHERE_DATACENTER="SDDC-Datacenter"                                     # The vSphere datacenter to deploy the management cluster on
+export VSPHERE_DATASTORE="DefaultDatastore"                                     # The vSphere datastore to deploy the management cluster on
+export VSPHERE_NETWORK="VM Network"                                             # The VM network to deploy the management cluster on
+export VSPHERE_RESOURCE_POOL="*/Resources"                                      # The vSphere resource pool for your VMs
+export VSPHERE_FOLDER="vm"                                                      # The VM folder for your VMs. Set to "" to use the root vSphere folder
+export VSPHERE_TEMPLATE="ubuntu-1804-kube-v1.17.3"                              # The VM template to use for your management cluster.
+export CONTROL_PLANE_ENDPOINT_IP="192.168.9.230"                                # the IP that kube-vip is going to use as a control plane endpoint
+export VIP_NETWORK_INTERFACE="ens192"                                           # The interface that kube-vip should apply the IP to. Omit to tell kube-vip to autodetect the interface.
+export VSPHERE_TLS_THUMBPRINT="..."                                             # sha1 thumbprint of the vcenter certificate: openssl x509 -sha1 -fingerprint -in ca.crt -noout
```

Comment:
same here, leave these unset, and add a `, e.g. "10.0.0.1"` etc at the end of the comments

Reply:
this is to prevent issues where this config is not set properly, and instead of a failure users get an invalid cluster template